examples/arithmetic.pln

```diff
@@ -0,0 +1,7 @@
+function add(a: any, b: any): integer
```

Comment:
Could you write a arithmetic example that shows a little more of Pallene? Something like factorial or Fibonacci would be nice.

Reply:
Yeah, sure. I'll add a few more examples shortly.

PS: The arithmetic.pln program was written to demonstrate the `any` keyword. I'll try to add comments which describe the purpose of each example.

Reply:
A example with any will make more sense if you use the recently-added `type` function.

Reply:
Where can I find the documentation for the type function? I am not able to find it in the manual.

Reply:
We could add this later.

Reply:
Yeah, alright.